### ISSUE TITLE
tmux 3.0 and 3.0a exits: [exited]

### ISSUE THREAD

**Issue by EdmundsEcho** [tmux/tmux#2004]
Tmux is a great app.  I use it close to 100% of the time.  Unfortunately, I ran into the following issue.

### Issue description
`tmux` exits right away.
`[exited]`

I have commented out all but my tmux key-binding in the `tmux.conf` file.  I also read through the updates and did not see anything obvious (other than to upgrade from 3.0 where I was also having trouble).  

Other info: I'm using iTerm2 `Build 3.3.7`

Thank you in advance.

Please read https://github.com/tmux/tmux/blob/master/.github/CONTRIBUTING.md
before opening an issue.

If you have upgraded, make sure your issue is not covered in the CHANGES file
for your version: https://raw.githubusercontent.com/tmux/tmux/master/CHANGES

Describe the problem and the steps to reproduce. Add a minimal tmux config if
necessary. Screenshots can be helpful, but no more than one or two.

Do not report bugs (crashes, incorrect behaviour) without reproducing on a tmux
built from the latest code in Git.

### Required information

Please provide the following information:

* tmux version (`tmux -V`).
3.0a

* Platform (`uname -sp`).
mac OS 10.15.1

* $TERM inside and outside of tmux (`echo $TERM`).
tmux-256color

* Logs from tmux (`tmux kill-server; tmux -vv new`).

```
1575493987.156564 client started (35566): version 3.0a, socket /private/tmp/tmux-501/default, protocol 8
1575493987.156627 on Darwin 19.0.0 Darwin Kernel Version 19.0.0: Thu Oct 17 16:17:15 PDT 2019; root:xnu-6153.41.3~29/RELEASE_X86_64; libevent 2.1.11-stable (select)
1575493987.156676 socket is /private/tmp/tmux-501/default
1575493987.156703 trying connect
1575493987.156819 add peer 0x7fb118000000: 6 (0x0)
1575493987.158106 sending message 100 to peer 0x7fb118000000 (4 bytes)
...
1575493987.158624 sending message 105 to peer 0x7fb118000000 (12 bytes)
1575493987.158638 sending message 105 to peer 0x7fb118000000 (14 bytes)
1575493987.158650 sending message 105 to peer 0x7fb118000000 (19 bytes)
1575493987.158660 sending message 105 to peer 0x7fb118000000 (38 bytes)
1575493987.158672 sending message 106 to peer 0x7fb118000000 (0 bytes)
1575493987.158685 sending message 200 to peer 0x7fb118000000 (4 bytes)
1575493987.158697 client loop enter
1575493987.160250 peer 0x7fb118000000 message 207
1575493987.160274 sending message 208 to peer 0x7fb118000000 (0 bytes)
1575493987.160314 peer 0x7fb118000000 message 203
1575493987.160330 sending message 205 to peer 0x7fb118000000 (0 bytes)
1575493987.160507 peer 0x7fb118000000 message 204
1575493987.160528 client loop exit
```

**Comment by nicm** [comment]
What is in the server log? It you have no server log, then you need to kill tmux first (pkill tmux).

**Comment by EdmundsEcho** [comment]
Thank you for your quick response. I was just updating the issue with the log.

[tmux-server-36655.log](https://github.com/tmux/tmux/files/3924027/tmux-server-36655.log)

**Comment by nicm** [comment]
It looks to me like your shell exited, what happens if you do this:

tmux kill-server
tmux new 'sleep 1000'

Does tmux start?

**Comment by EdmundsEcho** [comment]
I figured it out.  

My config file was linked to `~/dotfiles/` directory using a symlink.  This was a setup that worked for years.  When I installed the tmux v3.0 on Nov 30th, it deleted my configuration file.  The issue was hidden by the fact that for whatever reason, I have to run `tmux kill-server` to get around the exiting problem first discussed in this issue.  I know the file was deleted (at least "over-written" with an empty file), based on when the config file was last updated.  

The final data point.  Upon copying the config file from my `~/dotfiles/` to `~/.tmux.conf`, tmux behaves as expected (including letting me know that I had an outdated entry in my configuration).

Thank you for your help.
